catalog/secrets/api/src/main/java/org/projectnessie/catalog/secrets/UnsafePlainTextSecretsProvider.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2024 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.catalog.secrets;
+
+import jakarta.annotation.Nonnull;
+import java.net.URI;
+import java.util.Map;
+
+public class UnsafePlainTextSecretsProvider extends AbstractMapBasedSecretsProvider {
+  private final Map<URI, Map<String, String>> unsafeSecrets;
```

Comment:
Is the `Map` necessary with URNs? How about `urn:nessie-secret:plain:key1=value1,key2=value2`?

Reply:
Not sure what you're asking for. This class is rather just for testing purposes.
Also, there's by design no need to ask for multiple secrets.

Reply:
I mean putting plain test secrets directly in the URN and avoiding an extra config/map layer.